src/VirtoCommerce.Platform.Web/wwwroot/js/app/workspace.tpl.html

```diff
@@ -1,8 +1,11 @@
 <va-main-menu ng-model="mainMenu" on-menu-changed="onMainMenuChanged(mainMenu)">
-    <div class="logo">
-        <div class="logo-img">Virtocommerce</div>
+    <div class="logo" ng-if="!mainMenu.isCollapsed">
+        <img class="logo__img" ng-src="{{ uiCustomization.topPanelLogo_full_custom.url || uiCustomization.topPanelLogo_full_default.url }}" fallback-src="{{ uiCustomization.topPanelLogo_full_default.url }}" />
         <a class="logo-version" href="" ng-click="showVersionInfo()">{{platformVersion}}</a>
     </div>
+    <div class="logo" ng-if="mainMenu.isCollapsed">
+        <img class="logo__img logo__img--collapsed" ng-src="{{ uiCustomization.topPanelLogo_mini_custom.url || uiCustomization.topPanelLogo_mini_default.url }}" fallback-src="{{ uiCustomization.topPanelLogo_mini_default.url }}" />
+    </div>
```

Comment:
Does this `ng-src="x || y" fallback-src="y"` relative to comment below about falsy values support by fallbackSrc directive?

Reply:
Yes, it does.